[2025-09-10T06:21:11.932718] [INFO] [SYSTEM] Start new run (run_id=46f1ac08-f5be-4504-a9b5-12e95770a452)
[2025-09-10T06:21:35.986985] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 3
[2025-09-10T06:21:37.661068] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Counts computed for kit build.
[2025-09-10T06:21:43.046916] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Base64 length bounds computed for pad selection.
[2025-09-10T06:21:45.885302] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using PAD size (bucketized): 128
[2025-09-10T06:21:47.985535] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T06:21:48.544612] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T06:21:49.315831] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T06:21:49.316832] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Argon2id parameters confirmed for kit.
[2025-09-10T06:21:49.316832] [WARNING] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness below policy threshold.
[2025-09-10T06:21:52.105468] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness evaluation complete.
[2025-09-10T06:21:52.107468] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:21:52.414030] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:21:52.416032] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:21:52.484031] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:21:52.485031] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:21:53.068352] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:21:53.070353] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:21:53.657752] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:21:55.348283] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using user-provided password='Rex' (raw).
[2025-09-10T06:21:55.348283] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Starting Argon2id KDF (Noble). pass='Rex', salt(b64)='7p/7J+ALaAAkHOTgkBu+sg=='
[2025-09-10T06:21:55.460734] [ERROR] [GENERAL] debug_utils.py:log_exception:286 - Error during encrypted kit preparation
